hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -79,6 +81,7 @@
 import static org.apache.hadoop.fs.azurebfs.constants.HttpQueryParams.*;
 import org.apache.hadoop.fs.azurebfs.contracts.exceptions.AbfsRestOperationException;
```

Comment:
cut this line so the move of it to the right place, L59, doesn't upset checkstyle

Reply:
Not seeing this change in my local latest at all, that is there are no duplicate imports. Still, making the change to pass checkstyle.